mlflow/metrics/metric_definitions.py

```diff
@@ -16,20 +16,37 @@ def standard_aggregations(scores):
     }
 
 
-def _validate_text_data(data, metric_name, column_name):
-    """Validates that the data is text and is non-empty"""
-    if len(data) == 0:
-        return False
+def _is_text_data_invalid(data, metric_name, column_name):
+    """Validates that data list a list of strs and is non-empty"""
+    if data is None or len(data) == 0:
+        return True
 
     for row, line in enumerate(data):
         if not isinstance(line, str):
             _logger.warning(
                 f"Cannot calculate {metric_name} for non-string inputs. "
-                + f"Non-string found for {column_name} on row {row}. skipping metric logging."
+                f"Non-string found for {column_name} on row {row}. skipping metric logging."
```

Comment:
Can you revert all the reformatting changes to the unrelated parts in this PR and put them to a separate PR?
In general, it makes the current PR easier to review.